app/gateway/changelog.md

```diff
@@ -6,6 +6,263 @@ no_version: true
 <!-- vale off -->
 
 
+## 3.2.0.0
+**Release Date** 2023/02/15
+
+### Breaking changes and deprecations
```

Comment:
@lena-larionova let's make the Deprecations a separate section, please. Breaking changes are happening now while Deprecations are announcement of future changes.